rasa/shared/core/domain.py

```diff
@@ -442,14 +442,14 @@ def __init__(
         self._initialize_forms(forms)
 
         self.slots = slots
-        self.templates = templates
+        self.responses = responses
```

Comment:
watch out here cc @wochinge @yennycheung

Reply:
@yennycheung We should in my opinion have a  `templates` property which simply forwards to `responses` but raises a deprecation warning as the `Domain` is part of the Rasa API.

Reply:
You mean this endpoint?

Reply:
Yes, the endpoint, but also the Python API itself.

Reply:
Okay, I checked things on the RasaX side too, this looks good https://github.com/RasaHQ/rasa-x/blob/main/src/community/rasax/community/services/domain_service.py#L1139, shouldn't break anything.